tcp_sink.go

```diff
@@ -115,16 +108,61 @@ func (s *tcpStatsdSink) handleFlushError(err error, droppedBytes int) {
 	})
 }
 
+func (s *tcpStatsdSink) flushUint64(name, suffix string, u uint64) {
+	b := pbFree.Get().(*buffer)
```

Comment:
plz define a type-safe getter for use at all call-sites

Reply:
Would doing so really improve anything?  This is used in two places and `getBuffer()` is already defined, but does something else.

Reply:
k. no worries. sgtm